### ISSUE TITLE
Determine if curriculum is considered a public good

### ISSUE THREAD

**Issue by bevasq** [usds/ditap-curriculum-update#113]
**Dependencies**
- Should the curriculum be considered a public good? And would management be able to accommodate?

**Definition of success**
- Decision is made regarding curriculum as a public good and documented

**Other information**
- https://docs.google.com/document/d/1-OPQBWXMbH4SmqSu9iSkv0mBU4RS2Uovuatg2SQI2qg/edit?tab=t.0

**Additional context**

**Comment by bevasq** [comment]
@april-cruz does this ticket really belong under WS1 (vs WS2)?

**Comment by april-cruz** [comment]
@bevasq Yes, this ticket should actually be WS1. Mike will be helping us to figure out what this looks like and we can report out on our discussion today and future ones so you all are aware of progress/dependencies.
